projects/configs/tag-operations/base.yaml

```diff
@@ -0,0 +1,496 @@
+#https://dicom.innolitics.com/ciods/breast-projection-x-ray-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/enhanced-continuous-rt-image/general-equipment/00181020
+#NOTE The search of "00181020" found 579 results
+- name: "Focal Spot"
+  group: 0x0018
+  element: 0x1190
+  op: "keep"
+#https://dicom.innolitics.com/ciods/ct-image/ct-image/00181190
+#https://dicom.innolitics.com/ciods/ct-image/ct-image/00189360/00181190
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/x-ray-acquisition/00181190
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/x-ray-acquisition/00181190
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/x-ray-generation/00181190
+#https://dicom.innolitics.com/ciods/digital-mammography-x-ray-image/x-ray-generation/00181190
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/x-ray-generation/00181190
+#https://dicom.innolitics.com/ciods/breast-projection-x-ray-image/enhanced-mammography-image/00181190
+#https://dicom.innolitics.com/ciods/x-ray-3d-angiographic-image/x-ray-3d-angiographic-acquisition/00189507/00181190
+- name: "Patient Position"
+  group: 0x0018
+  element: 0x5100
+  op: "replace"
+#https://dicom.innolitics.com/ciods/ultrasound-image/general-series/00185100
+#https://dicom.innolitics.com/ciods/mr-image/general-series/00185100
+#https://dicom.innolitics.com/ciods/ct-image/general-series/00185100
+#NOTE: Seach term: "00185100 Image General Series"
+#################################### 0020 Group ###################################
+#
+#
+- name: "Study Instance UID"
+  group: 0x0020
+  element: 0x000d
+  op: "replace_UID"
+#https://dicom.innolitics.com/ciods/ultrasound-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/mr-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/ct-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/rt-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/digital-mammography-x-ray-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/vl-slide-coordinates-microscopic-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/x-ray-3d-craniofacial-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/x-ray-3d-angiographic-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/breast-projection-x-ray-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/legacy-converted-enhanced-pet-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/enhanced-rt-image/general-study/0020000d
+#https://dicom.innolitics.com/ciods/enhanced-continuous-rt-image/general-study/0020000d
```

Comment:
Can you slim all of these comments down to how you've have it later on listing the modalities that it covers 🙏🏻